modules/claims_api/app/services/claims_api/disability_compensation/mock_aws_service.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module V2
+    class MockAwsService
+
+      def sleep_interval
+        Settings.claims_api.mock_aws_service.sleep_interval
```

Comment:
are we looking to modify settings values specifically for this or should we just hardcode a range in here that will be randomized?